### ISSUE TITLE
Beoplay Input at startup either "Input 1" or "Join Group" not working

### ISSUE THREAD

**Issue by winzilla** [connectjunkie/homebridge-beoplay#31]
Thanks for all the great work, solved a lot of my issues! Except for one (but might also be me not configuring the plugin properly):
- Powering on the first B&O speaker shall use input 1 (which is TuneIn in my case)
- All other speakers shall use the "join" command when being powered-up (i.e. each speaker to check if another is already running and if so, join, else use input 1).
- In my case when configured to use the "join" command, first speaker always uses input "None" and subsequently all other speakers join the "None" input

**Comment by connectjunkie** [comment]
@winzilla Did you ever get this working how you would expect?  I only have the one Beoplay speaker, so haven't been able to test this behaviour.

**Comment by winzilla** [comment]
@connectjunkie No, saddly not, I also never managed to omit unused sources. But I‘m glad the plugin exists at all
